<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetSettings</class>
 <widget class="QTabWidget" name="WidgetSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>330</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TabWidget</string>
  </property>
  <widget class="QWidget" name="tab">
   <attribute name="title">
    <string>View Settings</string>
   </attribute>
   <widget class="QGroupBox" name="GridSettings">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>171</width>
      <height>241</height>
     </rect>
    </property>
    <property name="title">
     <string>Grid Settings</string>
    </property>
    <widget class="QRadioButton" name="en_pointGrid">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>95</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Point Grid</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="en_lineGrid">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>95</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Line Grid</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>90</y>
       <width>171</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButtoncloseSettings">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>260</y>
      <width>101</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Apply and Close</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="tab2">
   <attribute name="title">
    <string>Tab 2</string>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
